exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/ColumnType.kt

```diff
@@ -59,6 +59,25 @@ interface IColumnType {
     /** Returns the SQL representation of the specified **non-null** [value], for this column type. */
     fun nonNullValueToString(value: Any): String = notNullValueToDB(value).toString()
 
+    /**
+     * Returns the String representation of the specified [value] when [value] is set as the default for
+     * the column.
+     * If the value is `null` and the column is not nullable, an exception will be thrown.
+     */
+    fun valueAsDefaultString(value: Any?): String = when (value) {
```

Comment:
It would be great to explicitly document the difference between `valueToString` and other functions in `valueToString` KDoc (especially about cases when they should be used)